packages/admin/admin/src/common/buttons/split/SplitButton.tsx

```diff
@@ -71,22 +73,36 @@ const SplitBtn = ({
 
     const ActiveChild = childrenArray[_selectedIndex] as React.ReactElement;
 
-    const { variant: activeChildVariant, color: activeChildColor } = ActiveChild.props;
+    const { variant: activeChildVariant, color: activeChildColor, hasConflict: activeChildHasConflict } = ActiveChild.props;
+
+    const color = activeChildHasConflict ? "error" : activeChildColor;
 
     const showSelect = showSelectButtonState != null ? showSelectButtonState : showSelectButton;
     return (
         <SplitButtonContext.Provider value={{ setShowSelectButton: setShowSelectButtonState }}>
-            <ButtonGroup variant={activeChildVariant} color={activeChildColor} {...restProps} ref={anchorRef}>
+            <ButtonGroup
+                variant={activeChildVariant}
+                color={color}
+                sx={
+                    activeChildHasConflict
```

Comment:
Previously, the SplitButton didn't know anything about the states of the SaveSplitButton. I'd prefer if we can keep it this way. One way to achieve this would be to allow passing the `sx` prop of the `ActiveChild` to the button, and move the conflict styling into the SaveSplitButton. What do you think?

Reply:
The SplitButton is "stupid" again